app/src/main/java/com/mensinator/app/settings/SettingsViewModel.kt

```diff
@@ -56,6 +56,7 @@ class SettingsViewModel(
             defaultImportFilePath = exportImport.getDefaultImportFilePath(),
 
             showFaqDialog = false,
+            showLutealWarningDialog = false,
```

Comment:
Please change the order, showLutealWarningDialog should come before showFaqDialog